run_whole.py

```diff
@@ -1,61 +0,0 @@
-from cmdline import args
```

Comment:
why was the file deleted?

Reply:
because I am trying to be neat with my PRs. It is included in my latest PR

Reply:
But this is an existing file that we used for the benchmarks...

Reply:
it is? it shouldn't be....